views/welcome.html

```diff
@@ -83,8 +84,23 @@ <h2>Integrations</h2>
           </div>
         </div>
       </div>
-      
-      
+
+
+      <div class="tab-pane fade show" id="list-create-azure-host-policy" role="tabpanel" aria-labelledby="list-create-azure-host-policy-list">
+        <div class="row">
+          <div class="col-6">
+            <Br/>
+            <h2>Azure Authenticator</h2>
+            Generate a policy defining a host that can authenticate with the Azure Authenticator, using an Azure AD token
```

Comment:
We don't support Azure AD token but Azure managed identity token. It's not the same
You actually have two different Azure AD tokens - B2B token and B2C token... we support neither